### PR TITLE
Require file scope namespaces

### LINKED ISSUE #1892: Enable ImplicitUsings across whole repository
I suggest adding `<ImplicitUsings>enable</ImplicitUsings>` in `csproj`.

_Originally posted by @pellared in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1889#discussion_r1061311820_

### PR DESCRIPTION
**Pull request by RassK**
## Why

Fixes #1892 

## What

Requires file scoped namespaces

(Currently excludes examples, do we want to force it there too?)

## Tests

existing

### CODE REVIEW

File: test/test-applications/integrations/Directory.Build.props [hunk 1]
```diff
@@ -2,9 +2,13 @@
   <Import Project="$(MSBuildThisFileDirectory)..\..\..\Common.props" />
```
Comment (Kielek) [inline test/test-applications/integrations/Directory.Build.props:2]: I would put `<ImplicitUsings>enable</ImplicitUsings>` to Common.props.

**Review by Kielek — COMMENTED** [review]
I think that you could make using cleanups after this change.

BTW PR descriptions is wrong :)

File: test/test-applications/integrations/Directory.Build.props [hunk 2]
```diff
@@ -2,9 +2,13 @@
...
 
   <Import Project="$(MSBuildThisFileDirectory)Integrations.props" />
+
+  <PropertyGroup>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
   
   <!-- StyleCop -->
-  <ItemGroup>
+  <ItemGroup>
```
Comment (pellared) [inline test/test-applications/integrations/Directory.Build.props:11]:
```suggestion
  <ItemGroup>
```

**Review by pellared — COMMENTED** [review]
> Require file scope namespaces 

Is the title/description OK?

File: test/test-applications/integrations/Directory.Build.props (hunk 1 above)
Comment (RassK) [reply test/test-applications/integrations/Directory.Build.props:2]: thanks, didn't notice that one 👍

**Comment by RassK** [comment]
> Is the title/description OK?

I don't see why its misleading? It's causing IDE0161 which is treated as an error and explicitly saying "Convert to file-scoped namespace".

**Comment by Kielek** [comment]
> > Is the title/description OK?
> 
> I don't see why its misleading? It's causing IDE0161 which is treated as an error and explicitly saying "Convert to file-scoped namespace".

https://www.hanselman.com/blog/implicit-usings-in-net-6 vs https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces

**Comment by RassK** [comment]
> > > Is the title/description OK?
> > 
> > 
> > I don't see why its misleading? It's causing IDE0161 which is treated as an error and explicitly saying "Convert to file-scoped namespace".
> 
> https://www.hanselman.com/blog/implicit-usings-in-net-6 vs https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces

Oh nvm 😆 ... Different feature entirely. Closing, will create a new PR.

**Pull request closed without merging**